src/kava.ts

```diff
@@ -326,6 +326,7 @@ class Kava extends BasePlugin {
     this.eventManager.listen(this.player, this.player.Event.Core.MUTE_CHANGE, () => this._updateSoundModeInModel());
     this.eventManager.listen(this.player, this.player.Event.Core.ENTER_FULLSCREEN, () => this._onFullScreenChanged(ScreenMode.FULLSCREEN));
     this.eventManager.listen(this.player, this.player.Event.Core.EXIT_FULLSCREEN, () => this._onFullScreenChanged(ScreenMode.NOT_IN_FULLSCREEN));
+    this.eventManager.listen(this.player, this.player.Event.REGISTERED_PLUGINS_LIST_EVENT, (e) => this._onRegisteredPluginsListChange(e.payload));
```

Comment:
the event fires when the active plugins list is updated ?
maybe it should be called PLUGIN_LIST_UPDATED then ?

Reply:
The main goal of this event is not notifying of plugins list change but just to expose the plugins list (on player load)
But anyway the kaltura pr was already merged 
If it is critical I can open separate pr for this in kaltura player